src/test/java/org/jabref/logic/bibtexkeypattern/MakeLabelWithoutDatabaseTest.java

```diff
@@ -0,0 +1,46 @@
+package org.jabref.logic.bibtexkeypattern;
+
+import org.jabref.model.entry.BibEntry;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MakeLabelWithoutDatabaseTest {
+
+    // private GlobalBibtexKeyPattern pattern;
+    private BibEntry entry;
+
+    @Before
+    public void setUp() {
+        entry = new BibEntry();
+        entry.setField("author", "John Doe");
+        entry.setField("year", "2016");
+        entry.setField("title", "An awesome paper on JabRef");
+    }
+
+    @Test
+    public void makeLabelForFileSearch() {
+        String label = BibtexKeyPatternUtil.makeLabel(entry,
+                /*value=*/ "auth",
```

Comment:
Hmmm, indicating “Named Function Parameter” Comments is a recommended practice from the Dustin Boswell, Trevor Foucher "The Art of Readable Code", for languages that do not support named parameters, and I personally find such code easier to read, with not so much overhead. But if this is not used on JabRef and you do not find them useful, then, sure, please just remove them. Should I make the requested changes and recommit, or will you andle these issues?